startup/gui/layouts.py

```diff
@@ -66,10 +67,10 @@
+layouts.add( 'Scene', "GafferUI.CompoundEditor( scriptNode, _state={ 'children' : ( GafferUI.SplitContainer.Orientation.Horizontal, 0.772298, ( ( GafferUI.SplitContainer.Orientation.Horizontal, 0.256131, ( {'tabs': (GafferSceneUI.HierarchyView( scriptNode ), GafferSceneUI.SetEditor( scriptNode )), 'currentTab': 0, 'tabsVisible': True}, ( GafferUI.SplitContainer.Orientation.Vertical, 0.501064, ( ( GafferUI.SplitContainer.Orientation.Vertical, 0.927195, ( {'tabs': (GafferUI.Viewer( scriptNode ), GafferSceneUI.UVInspector( scriptNode )), 'currentTab': 0, 'tabsVisible': True}, {'tabs': (GafferUI.Timeline( scriptNode ),), 'currentTab': 0, 'tabsVisible': False} ) ), {'tabs': (GafferUI.GraphEditor( scriptNode ), GafferSceneUI.LightEditor( scriptNode ), GafferUI.AnimationEditor( scriptNode ), GafferSceneUI.PrimitiveInspector( scriptNode )), 'currentTab': 0, 'tabsVisible': True} ) ) ) ), ( GafferUI.SplitContainer.Orientation.Vertical, 0.501064, ( {'tabs': (GafferUI.NodeEditor( scriptNode ),), 'currentTab': 0, 'tabsVisible': True}, {'tabs': (GafferSceneUI.SceneInspector( scriptNode ),), 'currentTab': 0, 'tabsVisible': True} ) ) ) ), 'detachedPanels' : (), 'windowState' : { 'screen' : -1, 'fullScreen' : False, 'maximized' : True, 'bound' : imath.Box2f( imath.V2f( 0, 0.317554235 ), imath.V2f( 0.666666687, 1.0276134 ) ) }, 'editorState' : {'c-0-0-0-0': {'nodeSet': 'scriptNode.focusSet()'}, 'c-0-0-0-1': {'nodeSet': 'scriptNode.focusSet()'}, 'c-0-1-0-0-0-0': {'nodeSet': 'scriptNode.focusSet()'}, 'c-0-1-0-0-0-1': {'nodeSet': 'scriptNode.focusSet()'}, 'c-0-1-1-0-1': {'nodeSet': 'scriptNode.focusSet()'}, 'c-0-1-1-0-3': {'nodeSet': 'scriptNode.focusSet()'}, 'c-1-1-0-0': {'nodeSet': 'scriptNode.focusSet()'}} } )" )
```

Comment:
Perhaps the SetEditor should be below rather than behind the HierarchyView in this layout? It seems like you'd often use them in tandem, so it's a pity to make that hard.

Reply:
Adjusted the layout in 0e60294. It does seem an improvement over hiding the SetEditor away in a tab behind the HierarchyView.